### DISCUSSION TITLE
Add 12V Wireless Relay Tamper entity

### DISCUSSION THREAD

**Discussion by bass225** [petrleocompel/hikaxpro_hacs#128] [Ideas]
Hi There, 

Please consider adding the tamper entity for the wireless relay from link below. currently only reflecting the actual relay switching on HA.

https://www.hikvision.com/africa/products/Alarm-Products/wireless-intrusion-alarm/ax-pro/ds-pm1-o1l-we/

**Comment by petrleocompel** [comment]
Can you describe what you mean by "tamper entity" ? This ? -> "An alarm input for tamper, NO, NC" ? 

I am unsure. You can help with providing logs if you have the device. Since I do not own that device it is not possible without others cooperation.

**Reply by bass225** [reply]
The unit has one relay output and one tamper input. In my case its used on the electric fence, using the relay contact to switch it on and of and the tamper input to detect when alarm is triggered on the fence
![{4F75CB22-C77B-4972-AB0A-60110EC0B753}](https://github.com/user-attachments/assets/51c0881d-e75a-409c-b11f-edde258a7ab5)
![{4C34FB71-AC80-47A7-AFF6-1B27A9DDB801}](https://github.com/user-attachments/assets/8894087b-ce21-4e28-8c65-78ffa112d9e9)

Please let me know if you need any other info and how i can get it?

**Reply by petrleocompel** [reply]
I need logs from the integration as a start it will give me basic info of what the device is sending.
